src/aws_credentials_ecs.erl

```diff
@@ -21,7 +21,7 @@ fetch(_Options) ->
        , <<"SecretAccessKey">> := SecretAccessKey
        , <<"Token">> := Token
        , <<"Expiration">> := Expiration
-       } = jsx:decode(Body),
+       } = jsx:decode(Body, [return_maps]),
```

Comment:
I'm pretty sure version `3.0.0` of `jsx` doesn't need the `return_maps` option, I tried it in the shell and the change log [here](https://github.com/talentdeficit/jsx/blob/main/CHANGES.md) includes a mention to this.

Reply:
Option removed.